internal/util/util.go

```diff
@@ -193,3 +193,35 @@ func Mount(source, target, fstype string, options []string) error {
 	dummyMount := mount.New("")
 	return dummyMount.Mount(source, target, fstype, options)
 }
+
+// MountOptionsAdd adds the `add` mount options to the `options` and returns a
+// new string. In case `add` is already present in the `options`, `add` is not
+// added again.
+func MountOptionsAdd(options string, add ...string) string {
+	opts := strings.Split(options, ",")
```

Comment:
Idle speculation: perhaps instead of pulling a string apart over and over, mount options could be a new type over a slice and then only converted to a string as the last step before use?

Reply:
Yes, but there is a weird mix of strings formatted mount options and string slices. Reworking this would be a great cleanup, but also needs a bit more time.